### ISSUE TITLE
The tsconfig include property is not supported

### ISSUE THREAD

**Issue by rabbitfang** [TypeStrong/grunt-ts#381]
The `include` option in the [tsconfig.json](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html) file is not read by `grunt-ts`. It basically has the same behavior as `filesGlob`, but is officially supported.

**Comment by evil-shrike** [comment]
+1
I've also encountered that the plugin cannot compile my project.
I have tsconfig.json with `include` and no `exclude`. I run `npm run tsc` to compile, where "tsc" is a npm script name:
```json
  "scripts": {
    "tsc": "tsc",
    "test": "grunt test"
  },
```
It works fine.
But compiling via the plugin is failing.
Gruntfile:
```
    ts: {
        default : {
            tsconfig: 'true'
        }
    }
```
TypeScript 2.1
I'm not keed to replicate tsconfig settings in Gruntfile.

**Comment by evil-shrike** [comment]
And even if I specify files in Grunt task via `src` optons while keeping `tsconfig` option, the plugin ignores them and compile everything
```json
			default : {
				tsconfig: 'tsconfig.json',
				src: [
					"src/**/*.ts",
					"src/**/.*.ts",
					"../../../Client/src/**/*.ts",
					"../../../Client/src/**/.*.ts"
				]
			}
```

**Comment by rabbitfang** [comment]
@evil-shrike As a workaround, you can update the definition to be:
```js
{
  ts: {
    default: {
      tsconfig: {
        tsconfig: 'path/to/tsconfig.json',
        passThrough: true,
        updateFiles: false,
      }
    }
  }
}
```

This causes grunt-ts to call tsc with just the tsconfig.json file (and no files to compile).

**Comment by nycdotnet** [comment]
This is implemented in 6.0.0-beta.6 on npm.  Sorry for the delay. Please give it a whirl and let me know if you hit any issues.

**Comment by nycdotnet** [comment]
I am going to close this issue as I believe it is resolved in the latest beta.  Please feel free to reopen/comment.  Thank you!
